langchain4j-dashscope/src/test/java/dev/langchain4j/model/dashscope/QwenChatModelIT.java

```diff
@@ -99,7 +94,9 @@ public void should_call_function_with_argument_then_answer(String modelName) {
         ToolSpecification hasArgToolSpec = ToolSpecification.builder()
                 .name(toolName)
                 .description("Query the weather of a specified city")
-                .addParameter("cityName", JsonSchemaProperty.STRING)
+                .parameters(JsonObjectSchema.builder()
```

Comment:
I would keep this IT with an old API to make sure it works for users that still use it

Reply:
> I would keep this IT with an old API to make sure it works for users that still use it

Are you suggesting that I should continue using addParameter("cityName", JsonSchemaProperty.STRING) in the test code?

Reply:
Yes, maybe not in all tests, but al least in a few ones. `addParameter()` (old API) and `parameters(JsonObjectSchema)` (new API) should both work untill 1.0 version. Please note that `QwenAiServicesWithToolsIT` already tests new API.

Reply:
Rolled back.